src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -119,6 +157,8 @@ def __call__(
         latents: Optional[torch.FloatTensor] = None,
         output_type: Optional[str] = "pil",
         return_dict: bool = True,
+        callback: Optional[Callable[[int, np.ndarray, torch.FloatTensor], None]] = None,
```

Comment:
```suggestion
        callback: Optional[Callable[[int, int, torch.FloatTensor], None]] = None,
```
I _think_ that `timestep` should always be an int. There's an upcoming PR https://github.com/huggingface/diffusers/pull/637#issue-1385265940 that enforces it, but I believe it should already be the case in `main` after #534 was merged, see https://github.com/huggingface/diffusers/commit/bd8df2da89d99f630e5aa2ddb8f8cb45456561f1#diff-b3769bd64cdb5e99671dacb68cf04841d44a3e3ed003fe231e44ce5fe234b75cR141.

In any case, I think it'd be better to just send the scalar rather than a numpy array.

Reply:
Okay understood. Will add this change.